deploy/helm/sumologic/conf/traces/traces.conf

```diff
@@ -0,0 +1,14 @@
+<source>
+    @type zipkin
+    port 9411
+</source>
+<filter tracing.**>
+    @type stdout
+</filter>
+<match tracing.**>
+    @type zipkin
+    endpoint "#{ENV['SUMO_ENDPOINT_TRACES']}"
+    content_type application/json
+    overwrite_content_type application/vnd.sumologic.zipkin
```

Comment:
`overwrite_content_type` -> `overwrite_content_type` or change this value in zipkin plugin